spec-tests/src/test/java/org/candlepin/spec/ContentResourceSpecTest.java

```diff
@@ -223,6 +230,44 @@ public void shouldFilterContentWithMismatchedArchitectureFromTheProduct() throws
         assertEquals("ppc64", actualArch);
     }
 
+    @Nested
+    @Isolated
+    @Execution(ExecutionMode.SAME_THREAD)
+    class GetPages {
+        @Test
+        public void shouldListContentPagedAndSorted() throws InterruptedException {
+            ApiClient adminClient = ApiClients.admin();
+
+            OwnerDTO owner = adminClient.owners().createOwner(Owners.random());
+            String ownerKey = owner.getKey();
+
+            // Ensure that there is data for this test.
+            // Most likely, there will already be data left from other test runs that will show up
+            IntStream.range(0, 5).forEach(entry -> {
+                adminClient.ownerContent().createContent(ownerKey, Contents.random());
+                // for timestamp separation
+                try {
+                    sleep(1000);
+                }
+                catch (InterruptedException ie) {
+                    throw new RuntimeException("Unable to sleep as expected");
+                }
```

Comment:
Can we remove the catch an throw and just keep the `throws InterruptedException` in the method signature?

Also, are the sleeps needed between each content creation or is a sleep needed before calling `adminClient.content().getContents`? If the sleep is needed before calling `adminClient.content().getContents` we could move the sleep after creating all  the content.

Reply:
1. The exception is in the stream operation and the method signature won't see it. I went ahead and dropped the 'throws'
2. I need a delay in between creations to ensure that the created contents each have a unique timestamp in MySql [in seconds]

Reply:
+1 Thanks for the explanation! That makes sense